_docs/tasks/configuring-mixer.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Configuring Mixer
+overview: Shows how to configure Mixer.
+          
+order: 38
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+{% capture aspectConfig %}{{home}}/docs/reference/api/mixer/aspects{% endcapture %}
+{% capture adapterConfig %}{{home}}/docs/reference/api/mixer/adapters{% endcapture %}
+{% capture mixerConfig %}{{home}}/docs/reference/api/mixer/mixer-config.html{% endcapture %}
+
+This task shows how to configure Mixer.
+
+## Before you begin
+
+* [Install Istio](./installing-istio.html) in your kubernetes
+  cluster.
+
+* Deploy the [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application.
+
+* Initialize the application version routing by running the following
+  commands:
+  
+  ```bash
+  istioctl create -f route-rule-all-v1.yaml
+  istioctl replace -f route-rule-reviews-v2-v3.yaml
+  ```
+
+## Overview
+
+Mixer configuration expresses system behavior by specifying three 
+key pieces of information: **what** action to take, **how** to take that action, and **when** to take that action.
+
+* **What action to take:** _Aspect_ configuration defines _what_ action to take. These actions include
```

Comment:
can we link to reference docs for Aspect here?  and for Descriptors, and adapters (and possibly selectors and subjects) below?